Packs/FreshworksFreshservice/Integrations/FreshworksFreshservice/FreshworksFreshservice.yml

```diff
@@ -0,0 +1,4191 @@
+      description: Ticket sub-category.
+      type: String
+    - contextPath: Freshservice.Ticket.item_category
+      description: Ticket item category.
+      type: String
+    - contextPath: Freshservice.Ticket.id
+      description: Ticket ID.
+      type: Number
+    - contextPath: Freshservice.Ticket.type
+      description: Ticket type.
+      type: String
+    - contextPath: Freshservice.Ticket.to_emails
+      description: Email addresses to which the ticket was originally sent.
+      type: String
+    - contextPath: Freshservice.Ticket.department_id
+      description: Ticket department ID.
+      type: Number
+    - contextPath: Freshservice.Ticket.is_escalated
+      description: Whether the ticket is escalated.
+      type: Boolean
+    - contextPath: Freshservice.Ticket.due_by
+      description: The timestamp that denotes when the ticket is due to be resolved.
+      type: Date
+    - contextPath: Freshservice.Ticket.fr_due_by
+      description: Indicates when the first response is due.
+      type: Date
+    - contextPath: Freshservice.Ticket.created_at
+      description: Ticket creation time.
+      type: Date
+    - contextPath: Freshservice.Ticket.updated_at
+      description: Ticket updated at.
+      type: Date
+  - arguments:
+    - description: The Ticket ID to delete (use freshservice-ticket-list to get ticket ID).
+      isArray: false
+      name: ticket_id
+      required: true
+    deprecated: false
+    description: Delete an existing Ticket in Freshservice.
+    execution: false
+    name: freshservice-ticket-delete
+  - arguments:
+    - description: 'Page number of paginated results. Minimum value: 1'
+      isArray: false
+      name: page
+      required: false
+    - description: The number of items per page.
+      isArray: false
+      name: page_size
+      required: false
+    - defaultValue: '50'
+      description: The maximum number of records to retrieve.
+      isArray: false
+      name: limit
+      required: false
+    - description: The Ticket ID (use freshservice-ticket-list to get ticket ID).
+      isArray: false
+      name: ticket_id
+      required: true
+    - description: The Ticket task ID.
+      isArray: false
+      name: task_id
+      required: false
+    deprecated: false
+    description: Retrieve tasks list (or a specific task) on a Ticket with the given ID from Freshservice.
+    execution: false
+    name: freshservice-ticket-task-list
+    outputs:
+    - contextPath: Freshservice.Ticket.Task.id
+      description: Task ID.
+      type: Number
+    - contextPath: Freshservice.Ticket.Task.agent_id
+      description: Task agent ID.
+      type: Number
+    - contextPath: Freshservice.Ticket.Task.status
+      description: Task status.
+      type: Number
+    - contextPath: Freshservice.Ticket.Task.due_date
+      description: Task due date.
+      type: Date
+    - contextPath: Freshservice.Ticket.Task.notify_before
+      description: Task notify before.
```

Comment:
Don't understand this description.
notify before what?  is it a date?  Is it an event?